examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
```

Comment:
The parcellations are provided with the FS distribution, so I guess we could use them? The problem is that this function is not exactly optimized to plot labels / rois: the boundaries will be blurred because of the interpolation of vertex values to face values. An easy way around this should be to use the median value instead of the mean. But I thought this would go into a separate function (something similar to plot_roi?) and thus separate example. 
I agree that the data that is used now not a great example, because it isn't a stat map either, but just some random volume of a time series. So maybe I should look into creating an example map from the nki dataset? Or would it be an option to just use some map from Neurovault for now?